packages/modules/src/modules/epics/ContributionsEpic.tsx

```diff
@@ -252,6 +258,87 @@ const EpicBody: React.FC<BodyProps> = ({
     );
 };
 
+// TODO: recode to make this sub-component's image more responsive
+interface BylineWithImageProps {
+    bylineWithImage: BylineWithImage;
+}
+
+const bylineWithImageContainer = css`
+    margin: 0;
+    padding: 0;
+    position: relative;
+    width: 100%;
+    height: 130px;
+`;
+
+const bylineCopyContainer = css`
+    width: 70%;
+    position: absolute;
+    bottom: 20px;
+    left: 0;
+`;
+
+const bylineImageContainer = css`
+    width: 30%;
+    height: 100%;
+    position: absolute;
+    top: 0;
+    right: 0;
+`;
+
+const bylineName = css`
+    ${body.medium({ fontWeight: 'bold' })};
+    margin: 0;
+`;
+
+const bylineDescription = css`
+    ${body.medium({ fontStyle: 'italic' })};
+    margin: 0;
+`;
+
+const bylineHeadshotImage = css`
+    height: 100%;
+    width: 100%;
+    object-fit: cover;
+`;
+
+const bylineBottomDecoration = css`
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    background-repeat: repeat-x;
+    background-position: top;
+    background-size: 1px calc(0.25rem * 4 + 1px);
+    height: calc(0.25rem * 4 + 1px);
+    background-image: repeating-linear-gradient(
+        to bottom,
+        #dcdcdc,
+        #dcdcdc 1px,
+        transparent 1px,
+        transparent 0.25rem
+    );
+`;
+
+const BylineWithHeadshot: React.FC<BylineWithImageProps> = ({
```

Comment:
There's so much extra css here that I think it's best to move this component to its own file. (this file was already getting too long before your PR!)

Reply:
Done